### DISCUSSION TITLE
Which version is LTS, also docs do not show version 11

### DISCUSSION THREAD

**Discussion by fasteddys** [PiranhaCMS/piranha.core#2060] [General]
Hello, this is very nice asp needs a good CMS, I was reading the help and did not see some items.. and 11 help page was missing

____________

- [ ] is version 11 LTS, please choose one so we know you are supporting long term support
- [ ] The doc page does not show any help for version 11
- [ ] do you have in place updates
- [ ] how can we keep our code separate, un-impacted, from the piranha code, is there a code gen for a seperate module or plugin
- [ ] do you have a market place for plugins?
- [ ] how to do theming, or do you have themes
- [ ] `Marketplace for add-ons` or modules, themes?
- [ ] adding new entities for e.g. team, matches, scheduling, calendar etc

____________

![image](https://github.com/PiranhaCMS/piranha.core/assets/96149622/33d210df-9484-49c4-8b48-d6b39c41b7f8)

**Comment by tidyui** [comment]
Hi there! Let me try to respond to your questions.

## Is version 11 LTS?
LTS versions are usually something that commercial vendors have. This is free, non-profit, open source software meaning that we support the *latest version* (in terms of patches, bug fixes and minor releases). This is how we normally do our versioning.

* We change major version when increase the .NET framework dependency. This usually happens when .NET release their LTS-release. When .NET releases their short term version we normally release a minor version with cross compilation for the LTS version and the latest version.
* We *only* release bug fixes and patches for the current release. This means all bug fixes from now on will be release for `11` until version `12` is released. 
* We don't to breaking changes in minor releases. If we want to remove or change anything in a minor release we will keep it for compatibility and mark it with the `[Obsolete]` attribute that will produce compile warnings saying that this will be changed in the next major version.

## The doc page does not show any help for version 11
Thanks for the heads up, this was simply just forgotten. Luckily there are minimal API-changes between version `10` and `11`.

## Do you have in place updates?
It's up to you to update your application by updating the NuGet-packages.

## How can we keep our code separate, un-impacted, from the piranha code?
1. Use the available binary distribution from NuGet
2. Don't extend or modify Piranha's `DbContext`, instead add your own database entities in your own `DbContext`

## Do you have a market place for plugins?
Not at the moment. Piranha CMS is built for simple extensibility and it's completely decoupled or headless which doesn't lend itself well to Wordpress-style plugins.

## How to do theming, or do you have themes?
No. Piranha CMS is completely decoupled and doesn't care or know about the website rendering the application. The files your get when creating a project with our templates, for example `dotnet new piranha.mvc` are just templates and can be changed, replaced or removed as you see fit.

## Marketplace for add-ons or modules, themes?
No, see answer to the above two questions.

## Adding new entities for e.g. team, matches, scheduling, calendar etc
Custom business entities should be handled by your own `DbContext`, however the Admin UI can be easily extended to host all of your custom admin views as well. An example of this is the repository `Piranha.AspNetCore.Identity` which is self contained and has all database tools as well as admin views to support this feature.

Best regards
